### PR TITLE
JWPlayer Targeting Module

### PR DESCRIPTION
**Pull request by karimMourra**
## Type of change
- [x] Feature

## Description of change
Adds the `jwplayerTargeting` module which allows Ad Bidders to access JW Player's Video Ad Targeting information.
The module can be used as a submodule for prebid adapters, allowing them to use our `getTargetingForBid()` function.
the module will fetch segments for the media ids present in the prebid config when the module loads. If any bid requests are made while the segments are being fetched, they will be blocked until all requests complete, or the 150ms timeout expires.

### CODE REVIEW

**Comment by patmmccann** [comment]
Why build a new module type instead of utilizing an rtd submodule?

**Comment by patmmccann** [comment]
delaying the auction for data seems like it should use the rtd module https://github.com/prebid/Prebid.js/issues/4610 and your functionality feels like a submodule. A bunch of unicorn modules that don't fall into module types with rules described on https://github.com/prebid/Prebid.js/issues/5239 feels like potentially an end run around the rules at worst, duplicate functionality at best.

File: modules/jwplayerTargeting.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  if (segments) {
+    return segments;
+  }
+
+  const player = getPlayer(playerID);
+  if (!player) {
+    return [];
+  }
+
+  const item = mediaID ? player.getPlaylist().find(item => item.mediaid === mediaID) : player.getPlaylistItem();
```
Comment (patmmccann) [inline modules/jwplayerTargeting.js:93]: doesn't work on ie11

**Comment by patmmccann** [comment]
@omerBrowsi what are your thoughts on this

**Comment by karimMourra** [comment]
@patmmccann thank you for your feedback. Upon review of the rtdModule I think it does make sense for the JW Player module to be a submodule. I will close this PR for now. @omerBrowsi still looking forward to any feedback you may have.

**Pull request closed without merging**